browser/src/Services/Learning/Tutorial/Tutorials/SearchInBufferTutorial.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * TutorialManager
```

Comment:
I'm not super familiar with this codebase (having just started), but it seems like this comment is supposed to be  
```
/**
 * {filename}
 *
 * {short description}
 */

```

Reply:
Ah sorry I missed this @robin-pham - yep, that's the desired format for sure! 

I think unfortunately @muream inherited my mistake, some of the other tutorials I wrote have this problem too... sorry @muream!

Reply:
Haha, yep, I straight up copied the basic movement tutorial and built up from there.
I'll make sure to check for this kind of things going forward!